src/biogeochem/CNVegNitrogenFluxType.F90

```diff
@@ -164,7 +164,10 @@ module CNVegNitrogenFluxType
      real(r8), pointer :: deadstemn_storage_to_xfer_patch           (:)     ! patch dead stem N shift storage to transfer (gN/m2/s)
      real(r8), pointer :: livecrootn_storage_to_xfer_patch          (:)     ! patch live coarse root N shift storage to transfer (gN/m2/s)
      real(r8), pointer :: deadcrootn_storage_to_xfer_patch          (:)     ! patch dead coarse root N shift storage to transfer (gN/m2/s)
-     real(r8), pointer :: fert_patch                                (:)     ! patch applied fertilizer (gN/m2/s)
+     real(r8), pointer :: synthfert_patch                           (:)     ! patch applied synthetic fertilizer (gN/m2/s)
+     real(r8), pointer :: manure_patch                              (:)     ! patch applied manure (gN/m2/s)
+     real(r8), pointer :: nfertilization_patch                      (:)     ! patch applied total (synth. + manure) fertilizer (gN/m2/s)
+     
```

Comment:
- [ ] @juliusvira what happens to these variables when FAN is off?  Sorry, I've gotten lost in the code review process...?

Reply:
CropPhenology updates synthfert_patch and manure_patch regardless of FAN setting. If FAN is off or not coupled, then nfertilization becomes synthfert_patch + nfertilization_patch in CNNFert (within the CNNDynamics module).